Configuration.mk

```diff
@@ -73,12 +77,29 @@ TOCK_ARCHS := $(sort $(foreach target, $(TOCK_TARGETS), $(firstword $(subst |, ,
 
 # Check if elf2tab exists, if not, install it using cargo.
 ELF2TAB ?= elf2tab
+ELF2TAB_REQUIRED_VERSION := 0.7.0
 ELF2TAB_EXISTS := $(shell $(SHELL) -c "command -v $(ELF2TAB)")
+ELF2TAB_VERSION := $(shell $(SHELL) -c "$(ELF2TAB) --version | cut -d ' ' -f 2")
+
+# Check elf2tab version
+UPGRADE_ELF2TAB := $(shell $(SHELL) -c "printf '%s\n%s\n' '$(ELF2TAB_REQUIRED_VERSION)' '$(ELF2TAB_VERSION)' | sort --check=quiet --version-sort || echo yes")
+
+ifeq ($(UPGRADE_ELF2TAB),yes)
+  $(info Trying to update elf2tab to >= $(ELF2TAB_REQUIRED_VERSION))
+  override undefine ELF2TAB_EXISTS
+endif
+
 ifndef ELF2TAB_EXISTS
   $(shell cargo install elf2tab)
+  # Check elf2tab version after install
```

Comment:
```suggestion
  # Check elf2tab version after install
  ELF2TAB_VERSION := $(shell $(SHELL) -c "$(ELF2TAB) --version | cut -d ' ' -f 2")
```

Reply:
Works for me with both of these changes. Confirmed by downgrading to 0.6.0 then running make